### PR TITLE
Thoms' changes to sine2020 section

### PR DESCRIPTION
**Pull request by AnthonyLim23**
@thomasholmrod changes to SINE2020 section

### CODE REVIEW

File: guidelines_for_collaborative_development_of_sustainable_data_treatment_software.tex [hunk 1]
```diff
@@ -698,15 +698,78 @@ \subsection{Project Management Board and Advisory Committees}
 
 \todo{TODO: mention scientific coordination?}
 
-\section{SINE2020}
+\section{Software development for data treatment software for the neutron scattering and muon spectroscopy}
 \label{SINE2020}
 
-This section describes guidelines specific to the development of data treatment software for the neutron and muon communities.
-The specific guidelines in this section may not be suitable for other communities.
```
Comment (AnthonyLim23) [inline guidelines_for_collaborative_development_of_sustainable_data_treatment_software.tex]: I think this is important to say

File: guidelines_for_collaborative_development_of_sustainable_data_treatment_software.tex [hunk 2]
```diff
@@ -698,15 +698,78 @@ \subsection{Project Management Board and Advisory Committees}
...
 
-These specific guidelines were developed as part of the data treatment work package (WP 10) of Science and Innovation with Neutrons in Europe in 2020 (SINE2020).
-This was a 4-year project between 18 partner institutions spread across 12 different countries.
-The first deliverable (D10.1) was to write the standards and guidelines for software developed as part of SINE2020, with the aim of developing collaborative and interoperable software.
+These specific guidelines were developed as part of the data treatment work package (WP 10) in SINE2020.
+SINE2020 was a 4-year project between 18 partner institutions spread across 12 different countries. 
+Seven of the partners contributed to WP10 comprising six European neutron scattering and muon spectroscopy facilities (ESS, FZJ, LLB, ILL, ISIS, PSI) and University of Parma.
```
Comment (AnthonyLim23) [inline guidelines_for_collaborative_development_of_sustainable_data_treatment_software.tex]:
```suggestion
Seven of the partners contributed to WP10 comprising of the university or Parma and six European neutron scattering and muon spectroscopy facilities (ESS, FZJ, LLB, ILL, ISIS, and PSI). 
```

File: guidelines_for_collaborative_development_of_sustainable_data_treatment_software.tex [hunk 3]
```diff
@@ -698,15 +698,78 @@ \subsection{Project Management Board and Advisory Committees}
...
-The specific guidelines in this section may not be suitable for other communities.
 
-These specific guidelines were developed as part of the data treatment work package (WP 10) of Science and Innovation with Neutrons in Europe in 2020 (SINE2020).
-This was a 4-year project between 18 partner institutions spread across 12 different countries.
-The first deliverable (D10.1) was to write the standards and guidelines for software developed as part of SINE2020, with the aim of developing collaborative and interoperable software.
+These specific guidelines were developed as part of the data treatment work package (WP 10) in SINE2020.
+SINE2020 was a 4-year project between 18 partner institutions spread across 12 different countries. 
+Seven of the partners contributed to WP10 comprising six European neutron scattering and muon spectroscopy facilities (ESS, FZJ, LLB, ILL, ISIS, PSI) and University of Parma. 
+Technical University of Delft and Technical University of Munich, which both operates a research reactor as well as the HZB research reactor and Technical University of Denmark participated as observers in the work package.
```
Comment (AnthonyLim23) [inline guidelines_for_collaborative_development_of_sustainable_data_treatment_software.tex]:
```suggestion
Observers of the work package included the Technical University of Delft, the Technical University of Munich and the Technical University of Denmark.   
```

File: guidelines_for_collaborative_development_of_sustainable_data_treatment_software.tex [hunk 4]
```diff
@@ -698,15 +698,78 @@ \subsection{Project Management Board and Advisory Committees}
...
-This was a 4-year project between 18 partner institutions spread across 12 different countries.
-The first deliverable (D10.1) was to write the standards and guidelines for software developed as part of SINE2020, with the aim of developing collaborative and interoperable software.
+These specific guidelines were developed as part of the data treatment work package (WP 10) in SINE2020.
+SINE2020 was a 4-year project between 18 partner institutions spread across 12 different countries. 
+Seven of the partners contributed to WP10 comprising six European neutron scattering and muon spectroscopy facilities (ESS, FZJ, LLB, ILL, ISIS, PSI) and University of Parma. 
+Technical University of Delft and Technical University of Munich, which both operates a research reactor as well as the HZB research reactor and Technical University of Denmark participated as observers in the work package.   
+
+The first deliverable (D10.1) was to write the standards and guidelines for software developed in SINE2020 and the community as such with the aim of fostering development of sustainable, collaborative and interoperable software.
```
Comment (AnthonyLim23) [inline guidelines_for_collaborative_development_of_sustainable_data_treatment_software.tex]:
```suggestion
The first deliverable (D10.1) was to write a report on standards and guidelines for software development for
SINE2020. 
These guidelines were then used by the community to develop sustainable, collaborative and interoperable software. 
```

File: guidelines_for_collaborative_development_of_sustainable_data_treatment_software.tex [hunk 5]
```diff
@@ -698,15 +698,78 @@ \subsection{Project Management Board and Advisory Committees}
...
-The first deliverable (D10.1) was to write the standards and guidelines for software developed as part of SINE2020, with the aim of developing collaborative and interoperable software.
+These specific guidelines were developed as part of the data treatment work package (WP 10) in SINE2020.
+SINE2020 was a 4-year project between 18 partner institutions spread across 12 different countries. 
+Seven of the partners contributed to WP10 comprising six European neutron scattering and muon spectroscopy facilities (ESS, FZJ, LLB, ILL, ISIS, PSI) and University of Parma. 
+Technical University of Delft and Technical University of Munich, which both operates a research reactor as well as the HZB research reactor and Technical University of Denmark participated as observers in the work package.   
+
+The first deliverable (D10.1) was to write the standards and guidelines for software developed in SINE2020 and the community as such with the aim of fostering development of sustainable, collaborative and interoperable software. 
+To this end, the domain of scientific computing was considered in addition to an analysis of five programs considered in SINE2020.
```
Comment (AnthonyLim23) [inline guidelines_for_collaborative_development_of_sustainable_data_treatment_software.tex]: this does not add anything
```suggestion
```

File: guidelines_for_collaborative_development_of_sustainable_data_treatment_software.tex [hunk 6]
```diff
@@ -698,15 +698,78 @@ \subsection{Project Management Board and Advisory Committees}
...
+These specific guidelines were developed as part of the data treatment work package (WP 10) in SINE2020.
+SINE2020 was a 4-year project between 18 partner institutions spread across 12 different countries. 
+Seven of the partners contributed to WP10 comprising six European neutron scattering and muon spectroscopy facilities (ESS, FZJ, LLB, ILL, ISIS, PSI) and University of Parma. 
+Technical University of Delft and Technical University of Munich, which both operates a research reactor as well as the HZB research reactor and Technical University of Denmark participated as observers in the work package.   
+
+The first deliverable (D10.1) was to write the standards and guidelines for software developed in SINE2020 and the community as such with the aim of fostering development of sustainable, collaborative and interoperable software. 
+To this end, the domain of scientific computing was considered in addition to an analysis of five programs considered in SINE2020. 
+A brief description of those programs are given below followed by a discussion of specific guidelines for the neutron scattering and muon spectroscopy domain.
```
Comment (AnthonyLim23) [inline guidelines_for_collaborative_development_of_sustainable_data_treatment_software.tex]:
```suggestion
A brief description of the software programs are given below along with a discussion of specific guidelines for the neutron scattering and muon spectroscopy communities. 
```

File: guidelines_for_collaborative_development_of_sustainable_data_treatment_software.tex [hunk 7]
```diff
@@ -698,15 +698,78 @@ \subsection{Project Management Board and Advisory Committees}
...
+SINE2020 was a 4-year project between 18 partner institutions spread across 12 different countries. 
+Seven of the partners contributed to WP10 comprising six European neutron scattering and muon spectroscopy facilities (ESS, FZJ, LLB, ILL, ISIS, PSI) and University of Parma. 
+Technical University of Delft and Technical University of Munich, which both operates a research reactor as well as the HZB research reactor and Technical University of Denmark participated as observers in the work package.   
+
+The first deliverable (D10.1) was to write the standards and guidelines for software developed in SINE2020 and the community as such with the aim of fostering development of sustainable, collaborative and interoperable software. 
+To this end, the domain of scientific computing was considered in addition to an analysis of five programs considered in SINE2020. 
+A brief description of those programs are given below followed by a discussion of specific guidelines for the neutron scattering and muon spectroscopy domain. 
+
+\subsection{Analysed data treatment software for neutron scattering and muon spectroscopy}
```
Comment (AnthonyLim23) [inline guidelines_for_collaborative_development_of_sustainable_data_treatment_software.tex]: this is not a clear title

File: guidelines_for_collaborative_development_of_sustainable_data_treatment_software.tex [hunk 8]
```diff
@@ -698,15 +698,78 @@ \subsection{Project Management Board and Advisory Committees}
...
+The guidelines for data treatment software for the neutron scattering and muon spectroscopy communities are based on a questionnaire that five existing software projects have completed. Here we give a brief description of those software projects:
```
Comment (AnthonyLim23) [inline guidelines_for_collaborative_development_of_sustainable_data_treatment_software.tex]:
```suggestion
The original SINE2020 guidelines for data treatment software were based on a questionnaire, that was completed by five existing software projects.
```

File: guidelines_for_collaborative_development_of_sustainable_data_treatment_software.tex [hunk 9]
```diff
@@ -698,15 +698,78 @@ \subsection{Project Management Board and Advisory Committees}
...
+\begin{description}
+\item[BornAgain, www.bornagainproject.org] BornAgain started out as a reimplementation of the software XXX for simulation of a specific scattering technique grazing incidence Small Angle Scattering.
```
Comment (AnthonyLim23) [inline guidelines_for_collaborative_development_of_sustainable_data_treatment_software.tex]:
```suggestion
\item[BornAgain, www.bornagainproject.org] BornAgain provides a framework for calculating the scattering process from a multilayer sample, which may have a rough or smooth interface and can include the effect of nanoparticles.
It is a more generic version of GISAXS by R. Lazzari [J. Appl. Cryst. 35, 406–421 (2002)].
```

File: guidelines_for_collaborative_development_of_sustainable_data_treatment_software.tex [hunk 10]
```diff
@@ -698,15 +698,78 @@ \subsection{Project Management Board and Advisory Committees}
...
+In the course of SINE2020 it has been extended to other reflectometry techniques.
+The project started nine years ago and have around three dedicated full time developers.
```
Comment (AnthonyLim23) [inline guidelines_for_collaborative_development_of_sustainable_data_treatment_software.tex]:
```suggestion
The project started nine years ago and is supported by three full time developers. 
```

File: guidelines_for_collaborative_development_of_sustainable_data_treatment_software.tex [hunk 11]
```diff
@@ -698,15 +698,78 @@ \subsection{Project Management Board and Advisory Committees}
...
+\begin{description}
+\item[BornAgain, www.bornagainproject.org] BornAgain started out as a reimplementation of the software XXX for simulation of a specific scattering technique grazing incidence Small Angle Scattering.
+In the course of SINE2020 it has been extended to other reflectometry techniques.
+The project started nine years ago and have around three dedicated full time developers. 
+The software is primarily developed in C++ with a scripting interface in Python and a GUI based on Qt. The software is hosted on GitHub. 
+
+\item[Mantid, www.mantidproject.org] Mantid is a project initiated by ISIS eleven years ago and since then joined by SNS, ESS, and ILL, with the purpose of providing data reduction software for all their instruments. Besides these facilities Mantid is also being used for specific instruments at other neutron (and muon) sources such as SINQ, FRM-II, and ANSTO. Mantid is developed in C++ and Python with a GUI based on Qt.
```
Comment (AnthonyLim23) [inline guidelines_for_collaborative_development_of_sustainable_data_treatment_software.tex]:
```suggestion
\item[Mantid, www.mantidproject.org] Mantid is a project initiated by ISIS eleven years ago and now has contributions from the SNS, ESS, and ILL. 
The purpose of Mantid is to provide a single data reduction software for all of the instruments at each of the facilities.
Besides these facilities Mantid is also being used for specific instruments at other neutron (and muon) sources such as SINQ, FRM-II, and ANSTO. 
Mantid is developed using C++ and Python with a Qt based GUI. 
```

File: guidelines_for_collaborative_development_of_sustainable_data_treatment_software.tex [hunk 12]
```diff
@@ -698,15 +698,78 @@ \subsection{Project Management Board and Advisory Committees}
...
+\begin{description}
+\item[BornAgain, www.bornagainproject.org] BornAgain started out as a reimplementation of the software XXX for simulation of a specific scattering technique grazing incidence Small Angle Scattering.
+In the course of SINE2020 it has been extended to other reflectometry techniques.
+The project started nine years ago and have around three dedicated full time developers. 
+The software is primarily developed in C++ with a scripting interface in Python and a GUI based on Qt. The software is hosted on GitHub. 
+
+\item[Mantid, www.mantidproject.org] Mantid is a project initiated by ISIS eleven years ago and since then joined by SNS, ESS, and ILL, with the purpose of providing data reduction software for all their instruments. Besides these facilities Mantid is also being used for specific instruments at other neutron (and muon) sources such as SINQ, FRM-II, and ANSTO. Mantid is developed in C++ and Python with a GUI based on Qt. 
+The software is hosted on GitHub under the GPL3 license and the project has about 20 full time developers across all facilities involved.
```
Comment (AnthonyLim23) [inline guidelines_for_collaborative_development_of_sustainable_data_treatment_software.tex]:
```suggestion
The Mantid software is hosted on GitHub under the GPL3 license.
The project has about 20 full time developers spread across each of the partner facilities.
```

File: guidelines_for_collaborative_development_of_sustainable_data_treatment_software.tex [hunk 13]
```diff
@@ -698,15 +698,78 @@ \subsection{Project Management Board and Advisory Committees}
...
+It is by far the largest software project specific for data treatment at neutron and muon sources and has annual developer and user meetings, a XXX.
```
Comment (AnthonyLim23) [inline guidelines_for_collaborative_development_of_sustainable_data_treatment_software.tex]:
```suggestion
As one of the largest software projects for data treatment at neutron and muon sources, regular feedback is important.
Therefore, Mantid has annual user and developer meetings.
```

File: guidelines_for_collaborative_development_of_sustainable_data_treatment_software.tex [hunk 14]
```diff
@@ -698,15 +698,78 @@ \subsection{Project Management Board and Advisory Committees}
...
+
+\item[Mantid, www.mantidproject.org] Mantid is a project initiated by ISIS eleven years ago and since then joined by SNS, ESS, and ILL, with the purpose of providing data reduction software for all their instruments. Besides these facilities Mantid is also being used for specific instruments at other neutron (and muon) sources such as SINQ, FRM-II, and ANSTO. Mantid is developed in C++ and Python with a GUI based on Qt. 
+The software is hosted on GitHub under the GPL3 license and the project has about 20 full time developers across all facilities involved.
+It is by far the largest software project specific for data treatment at neutron and muon sources and has annual developer and user meetings, a XXX. 
+
+\item[KipTool \& MuhRec, www.neutronimaging.ch] KipTool and MuhRec are analysis software for the neutron imaging technique, where KipTool is used for quantitative analysis and MuhRec for computed tomography and has started as a spare time project for an instrument scientist who spent his time coding when commuting to and from work on the train.
```
Comment (AnthonyLim23) [inline guidelines_for_collaborative_development_of_sustainable_data_treatment_software.tex]:
```suggestion
\item[KipTool \& MuhRec, www.neutronimaging.ch] KipTool and MuhRec are analysis software for the neutron imaging technique.
KipTool is used for quantitative analysis of imaging data and MuhRec is used for computed tomography.
```

File: guidelines_for_collaborative_development_of_sustainable_data_treatment_software.tex [hunk 15]
```diff
@@ -698,15 +698,78 @@ \subsection{Project Management Board and Advisory Committees}
...
+
+\item[Mantid, www.mantidproject.org] Mantid is a project initiated by ISIS eleven years ago and since then joined by SNS, ESS, and ILL, with the purpose of providing data reduction software for all their instruments. Besides these facilities Mantid is also being used for specific instruments at other neutron (and muon) sources such as SINQ, FRM-II, and ANSTO. Mantid is developed in C++ and Python with a GUI based on Qt. 
+The software is hosted on GitHub under the GPL3 license and the project has about 20 full time developers across all facilities involved.
+It is by far the largest software project specific for data treatment at neutron and muon sources and has annual developer and user meetings, a XXX. 
+
+\item[KipTool \& MuhRec, www.neutronimaging.ch] KipTool and MuhRec are analysis software for the neutron imaging technique, where KipTool is used for quantitative analysis and MuhRec for computed tomography and has started as a spare time project for an instrument scientist who spent his time coding when commuting to and from work on the train. 
+Since then at most two full-time developers have been dedicated to the project at any given time.
```
Comment (AnthonyLim23) [inline guidelines_for_collaborative_development_of_sustainable_data_treatment_software.tex]: are MuhRec and KipTool one or two projects?

File: guidelines_for_collaborative_development_of_sustainable_data_treatment_software.tex [hunk 16]
```diff
@@ -698,15 +698,78 @@ \subsection{Project Management Board and Advisory Committees}
...
+Both software programs are originally implemented in C++ with the only user interface being in the form of a GUI based on Qt.
+In the course of the SINE2020 project, the software has been migrated to GitHub under the GPL3 license. 
+
+\item[McStas, www.mcstas.org] McStas is per se not a data treatment software but used for performing instrument simulations.
```
Comment (AnthonyLim23) [inline guidelines_for_collaborative_development_of_sustainable_data_treatment_software.tex]:
```suggestion
\item[McStas, www.mcstas.org] McStas is not a data treatment software, but it is used for performing instrument simulations.
```

File: guidelines_for_collaborative_development_of_sustainable_data_treatment_software.tex [hunk 17]
```diff
@@ -698,15 +698,78 @@ \subsection{Project Management Board and Advisory Committees}
...
+In was included in the analysis due to its widespread usage in the neutron scattering community.
```
Comment (AnthonyLim23) [inline guidelines_for_collaborative_development_of_sustainable_data_treatment_software.tex]:
```suggestion
These simulations can provide crucial insight into experiments when coupled with data treatment software. 
Hence, it has been included in this analysis as an important tool to the neutron scattering community. 
```

File: guidelines_for_collaborative_development_of_sustainable_data_treatment_software.tex [hunk 18]
```diff
@@ -698,15 +698,78 @@ \subsection{Project Management Board and Advisory Committees}
...
+
+\item[KipTool \& MuhRec, www.neutronimaging.ch] KipTool and MuhRec are analysis software for the neutron imaging technique, where KipTool is used for quantitative analysis and MuhRec for computed tomography and has started as a spare time project for an instrument scientist who spent his time coding when commuting to and from work on the train. 
+Since then at most two full-time developers have been dedicated to the project at any given time.
+Both software programs are originally implemented in C++ with the only user interface being in the form of a GUI based on Qt.
+In the course of the SINE2020 project, the software has been migrated to GitHub under the GPL3 license. 
+
+\item[McStas, www.mcstas.org] McStas is per se not a data treatment software but used for performing instrument simulations.
+In was included in the analysis due to its widespread usage in the neutron scattering community. 
+It is also the longest living among those analyzed with more than 20 years of effort behind it but only has one dedicated full-time developer associated to it.
```
Comment (AnthonyLim23) [inline guidelines_for_collaborative_development_of_sustainable_data_treatment_software.tex]:
```suggestion
McStats started more than 20 years ago and has continued to see new functionality being added. 
However, it only has one dedicated full-time developer. 
```

File: guidelines_for_collaborative_development_of_sustainable_data_treatment_software.tex [hunk 19]
```diff
@@ -698,15 +698,78 @@ \subsection{Project Management Board and Advisory Committees}
...
+Both software programs are originally implemented in C++ with the only user interface being in the form of a GUI based on Qt.
+In the course of the SINE2020 project, the software has been migrated to GitHub under the GPL3 license. 
+
+\item[McStas, www.mcstas.org] McStas is per se not a data treatment software but used for performing instrument simulations.
+In was included in the analysis due to its widespread usage in the neutron scattering community. 
+It is also the longest living among those analyzed with more than 20 years of effort behind it but only has one dedicated full-time developer associated to it. 
+McStas is developed primarily in C and has recently been migrated to GitHub.
+It is licensed under the GPL2 licence. 
+
+\item[SasView, www.sasview.org] SasView has only had dedicated full time developers when externally funded and has otherwise depended on the good will of scientists putting time into the project.
```
Comment (AnthonyLim23) [inline guidelines_for_collaborative_development_of_sustainable_data_treatment_software.tex]:
```suggestion
\item[SasView, www.sasview.org] SasView has been developed to be a simple and effective set of tools for the analysis of small angle neutron scattering data. 
```

File: guidelines_for_collaborative_development_of_sustainable_data_treatment_software.tex [hunk 20]
```diff
@@ -698,15 +698,78 @@ \subsection{Project Management Board and Advisory Committees}
...
+SasView has had at most two full time developers dedicated to it but has contributions from around fifteen developers from both photon and neutron sources, universities and other research institutions world wide.
```
Comment (AnthonyLim23) [inline guidelines_for_collaborative_development_of_sustainable_data_treatment_software.tex]:
```suggestion
SasView has had developers from both photon and neutron sources, with around fifteen contributors. 
Typically SasView has two full time developers, which is supplemented by additional developers at code camps (held once or twice a year).

had at most two full time developers dedicated to it but has contributions from around fifteen , universities and other research institutions world wide. 
```

File: guidelines_for_collaborative_development_of_sustainable_data_treatment_software.tex [hunk 21]
```diff
@@ -698,15 +698,78 @@ \subsection{Project Management Board and Advisory Committees}
...
+
+\item[McStas, www.mcstas.org] McStas is per se not a data treatment software but used for performing instrument simulations.
+In was included in the analysis due to its widespread usage in the neutron scattering community. 
+It is also the longest living among those analyzed with more than 20 years of effort behind it but only has one dedicated full-time developer associated to it. 
+McStas is developed primarily in C and has recently been migrated to GitHub.
+It is licensed under the GPL2 licence. 
+
+\item[SasView, www.sasview.org] SasView has only had dedicated full time developers when externally funded and has otherwise depended on the good will of scientists putting time into the project.
+SasView has had at most two full time developers dedicated to it but has contributions from around fifteen developers from both photon and neutron sources, universities and other research institutions world wide. 
+These developers often contribute during code camps held once or twice per year.
```
Comment (AnthonyLim23) [inline guidelines_for_collaborative_development_of_sustainable_data_treatment_software.tex]:
```suggestion
```

File: guidelines_for_collaborative_development_of_sustainable_data_treatment_software.tex [hunk 22]
```diff
@@ -698,15 +698,78 @@ \subsection{Project Management Board and Advisory Committees}
...
+\item[McStas, www.mcstas.org] McStas is per se not a data treatment software but used for performing instrument simulations.
+In was included in the analysis due to its widespread usage in the neutron scattering community. 
+It is also the longest living among those analyzed with more than 20 years of effort behind it but only has one dedicated full-time developer associated to it. 
+McStas is developed primarily in C and has recently been migrated to GitHub.
+It is licensed under the GPL2 licence. 
+
+\item[SasView, www.sasview.org] SasView has only had dedicated full time developers when externally funded and has otherwise depended on the good will of scientists putting time into the project.
+SasView has had at most two full time developers dedicated to it but has contributions from around fifteen developers from both photon and neutron sources, universities and other research institutions world wide. 
+These developers often contribute during code camps held once or twice per year. 
+To govern the project, SasView has a management team composed of an instrument scientist from each of the four neutron (and muon) sources: ESS, ISIS, NIST, and SNS.
```
Comment (AnthonyLim23) [inline guidelines_for_collaborative_development_of_sustainable_data_treatment_software.tex]:
```suggestion
To govern the project, SasView has a management team composed of an instrument scientist from each of the four neutron (and muon) sources (ESS, ISIS, NIST and SNS). 
```

File: guidelines_for_collaborative_development_of_sustainable_data_treatment_software.tex [hunk 23]
```diff
@@ -698,15 +698,78 @@ \subsection{Project Management Board and Advisory Committees}
...
+It is also the longest living among those analyzed with more than 20 years of effort behind it but only has one dedicated full-time developer associated to it. 
+McStas is developed primarily in C and has recently been migrated to GitHub.
+It is licensed under the GPL2 licence. 
+
+\item[SasView, www.sasview.org] SasView has only had dedicated full time developers when externally funded and has otherwise depended on the good will of scientists putting time into the project.
+SasView has had at most two full time developers dedicated to it but has contributions from around fifteen developers from both photon and neutron sources, universities and other research institutions world wide. 
+These developers often contribute during code camps held once or twice per year. 
+To govern the project, SasView has a management team composed of an instrument scientist from each of the four neutron (and muon) sources: ESS, ISIS, NIST, and SNS. 
+The software is developed in Python with some C and hosted on GitHub under the BSD license. 
+Prior to SINE2020, SasView had a GUI based on WX and no Python interface.
```
Comment (AnthonyLim23) [inline guidelines_for_collaborative_development_of_sustainable_data_treatment_software.tex]:
```suggestion
Prior to SINE2020, SasView had a GUI was based on WX and did not have a Python API.
```

Comment (AnthonyLim23) [reply guidelines_for_collaborative_development_of_sustainable_data_treatment_software.tex]: @thomasholmrod Im assuming you mean API when you say interface, so not to confuse it with GUI

File: guidelines_for_collaborative_development_of_sustainable_data_treatment_software.tex [hunk 24]
```diff
@@ -698,15 +698,78 @@ \subsection{Project Management Board and Advisory Committees}
...
+In the course of SINE2020, it has been equipped with a Python interface as well as a GUI based on Qt (from version 5).
```
Comment (AnthonyLim23) [inline guidelines_for_collaborative_development_of_sustainable_data_treatment_software.tex]:
```suggestion
In the course of SINE2020, it has been equipped with a Python API as well as a GUI based on Qt (from version 5). 
```

File: guidelines_for_collaborative_development_of_sustainable_data_treatment_software.tex [hunk 25]
```diff
@@ -725,25 +788,10 @@ \section{SINE2020}
...
- \end{tabular}
- \caption{An updated summary of how guidelines were applied to the different software projects from SINE200 \cite{} (TODO: ref paper). The number of guidelines implemented increases with the size
-and scope of the project. PMB = Project Management Board. AC = Advisory Committee}
-\label{summaryTable}
-\end{sidewaystable}
 
 The statistics of how guidelines could be applied to software projects are presented in table \ref{summaryTable}.
 Some of the projects have a large numbers of developers and/or based at multiple locations.
-As a result the management overheads (e.g. project management boards) are greater to ensure a uniform standard across the project.
+As a result the management overheads (e.g., project management boards) are greater to ensure a uniform standard across the project.
```
Comment (AnthonyLim23) [inline guidelines_for_collaborative_development_of_sustainable_data_treatment_software.tex]:
```suggestion
As a result the management overheads (e.g. project management boards) are greater to ensure a uniform standard across the project.
```

**Comment by AnthonyLim23** [comment]
@DanNixon Thomas has said that he has made some more changes since I made this PR. Shall we get this one merged (with our suggestions) and then I can make a new PR

**Comment by thomasholmrod** [comment]
As previously mentioned the SINE2020 section was work in progress and is now very different.

**Comment by thomasholmrod** [comment]
In the new version the software descriptions are removed. They can be included again, but most of the information is in the table anyway, so I think the description should be hold to a minimum, i.e. whay the software is used for.

File: guidelines_for_collaborative_development_of_sustainable_data_treatment_software.tex (hunk 1 above)
Comment (thomasholmrod) [reply guidelines_for_collaborative_development_of_sustainable_data_treatment_software.tex]: I think it is in the new new version

File: guidelines_for_collaborative_development_of_sustainable_data_treatment_software.tex (hunk 2 above)
Comment (thomasholmrod) [reply guidelines_for_collaborative_development_of_sustainable_data_treatment_software.tex]: In the new version all this is omitted. I find it annoying to read. On the other hand, it is also important to show how widely the community has been involved, so if somebody can find an elegant way of showing that would be useful.

Comment (thomasholmrod) [reply guidelines_for_collaborative_development_of_sustainable_data_treatment_software.tex]: committing change to this version

File: guidelines_for_collaborative_development_of_sustainable_data_treatment_software.tex (hunk 3 above)
Comment (thomasholmrod) [reply guidelines_for_collaborative_development_of_sustainable_data_treatment_software.tex]: Looks ok to me for this version, but again this has been removed from new version

File: guidelines_for_collaborative_development_of_sustainable_data_treatment_software.tex (hunk 4 above)
Comment (thomasholmrod) [reply guidelines_for_collaborative_development_of_sustainable_data_treatment_software.tex]: ok for this version

File: guidelines_for_collaborative_development_of_sustainable_data_treatment_software.tex (hunk 5 above)
Comment (thomasholmrod) [reply guidelines_for_collaborative_development_of_sustainable_data_treatment_software.tex]: agree. I think the point was supposed be to that the choices made reflects what is being used in scientific computing / data science more generally. In fact, some of the guidelines can be found already in the proposal. In the new version I just mention that many other programs (almost) adhere to the guidelines and add a few examples that needs to be checked.

File: guidelines_for_collaborative_development_of_sustainable_data_treatment_software.tex (hunk 6 above)
Comment (thomasholmrod) [reply guidelines_for_collaborative_development_of_sustainable_data_treatment_software.tex]: ok

File: guidelines_for_collaborative_development_of_sustainable_data_treatment_software.tex (hunk 7 above)
Comment (thomasholmrod) [reply guidelines_for_collaborative_development_of_sustainable_data_treatment_software.tex]: Perhaps not, in any case it does not fit to the new version. The new one is: "Guidelines for development of data treatment software for neutron scattering and muon spectroscopy" which is awfully long. Better ones are welcomed.

File: guidelines_for_collaborative_development_of_sustainable_data_treatment_software.tex (hunk 8 above)
Comment (thomasholmrod) [reply guidelines_for_collaborative_development_of_sustainable_data_treatment_software.tex]: ok for this version. Again, I am not sure that we need a description of the software at all.

File: guidelines_for_collaborative_development_of_sustainable_data_treatment_software.tex (hunk 9 above)
Comment (thomasholmrod) [reply guidelines_for_collaborative_development_of_sustainable_data_treatment_software.tex]: thanks!

File: guidelines_for_collaborative_development_of_sustainable_data_treatment_software.tex (hunk 10 above)
Comment (thomasholmrod) [reply guidelines_for_collaborative_development_of_sustainable_data_treatment_software.tex]: ok

File: guidelines_for_collaborative_development_of_sustainable_data_treatment_software.tex (hunk 11 above)
Comment (thomasholmrod) [reply guidelines_for_collaborative_development_of_sustainable_data_treatment_software.tex]: ok

File: guidelines_for_collaborative_development_of_sustainable_data_treatment_software.tex (hunk 12 above)
Comment (thomasholmrod) [reply guidelines_for_collaborative_development_of_sustainable_data_treatment_software.tex]: ok

File: guidelines_for_collaborative_development_of_sustainable_data_treatment_software.tex (hunk 13 above)
Comment (thomasholmrod) [reply guidelines_for_collaborative_development_of_sustainable_data_treatment_software.tex]: ok

File: guidelines_for_collaborative_development_of_sustainable_data_treatment_software.tex (hunk 14 above)
Comment (thomasholmrod) [reply guidelines_for_collaborative_development_of_sustainable_data_treatment_software.tex]: ok

File: guidelines_for_collaborative_development_of_sustainable_data_treatment_software.tex (hunk 15 above)
Comment (thomasholmrod) [reply guidelines_for_collaborative_development_of_sustainable_data_treatment_software.tex]: Well, to my understanding they are two different software programs but hosted in the same repository.  However, I don't see any specific kiptool releases (https://github.com/neutronimaging/imagingsuite). We can leave this for Anders K to sort out - I have a meeting with him on Friday.

File: guidelines_for_collaborative_development_of_sustainable_data_treatment_software.tex (hunk 16 above)
Comment (thomasholmrod) [reply guidelines_for_collaborative_development_of_sustainable_data_treatment_software.tex]: ok

File: guidelines_for_collaborative_development_of_sustainable_data_treatment_software.tex (hunk 17 above)
Comment (thomasholmrod) [reply guidelines_for_collaborative_development_of_sustainable_data_treatment_software.tex]: good

File: guidelines_for_collaborative_development_of_sustainable_data_treatment_software.tex (hunk 18 above)
Comment (thomasholmrod) [reply guidelines_for_collaborative_development_of_sustainable_data_treatment_software.tex]: ok

File: guidelines_for_collaborative_development_of_sustainable_data_treatment_software.tex (hunk 19 above)
Comment (thomasholmrod) [reply guidelines_for_collaborative_development_of_sustainable_data_treatment_software.tex]: ok

File: guidelines_for_collaborative_development_of_sustainable_data_treatment_software.tex (hunk 20 above)
Comment (thomasholmrod) [reply guidelines_for_collaborative_development_of_sustainable_data_treatment_software.tex]: ok

File: guidelines_for_collaborative_development_of_sustainable_data_treatment_software.tex (hunk 21 above)
Comment (thomasholmrod) [reply guidelines_for_collaborative_development_of_sustainable_data_treatment_software.tex]: This is perhaps the most important thing about SasView. I.e. that it for years has been kept alive by having these code camps and with almost nothing happening between. Anyway, removed in new version

File: guidelines_for_collaborative_development_of_sustainable_data_treatment_software.tex (hunk 22 above)
Comment (thomasholmrod) [reply guidelines_for_collaborative_development_of_sustainable_data_treatment_software.tex]: ok

File: guidelines_for_collaborative_development_of_sustainable_data_treatment_software.tex (hunk 23 above)
Comment (thomasholmrod) [reply guidelines_for_collaborative_development_of_sustainable_data_treatment_software.tex]: In essence it is both a user interface and an API.  Perhaps "Python scripting interface" would be better?

File: guidelines_for_collaborative_development_of_sustainable_data_treatment_software.tex (hunk 24 above)
Comment (thomasholmrod) [reply guidelines_for_collaborative_development_of_sustainable_data_treatment_software.tex]: see previous comment

File: guidelines_for_collaborative_development_of_sustainable_data_treatment_software.tex (hunk 25 above)
Comment (thomasholmrod) [reply guidelines_for_collaborative_development_of_sustainable_data_treatment_software.tex]: oh well, I guess there are different opinions about a ','  

_The consensus seems to be in favor of the comma in American usage; against it in British usage._ (https://www.dailywritingtips.com/comma-after-i-e-and-e-g/)

**Comment by thomasholmrod** [comment]
I suggest to just replace the SINE2020 version with the new one. It would most likely also benefit from some proof reading, but I think it is more to the point. In the new version focus is on the specific guidelines and why they are sensible. A paragraph about plotting modules has also been added although that has not been covered in SINE2020.

File: guidelines_for_collaborative_development_of_sustainable_data_treatment_software.tex [hunk 26]
```diff
@@ -698,15 +698,78 @@ \subsection{Project Management Board and Advisory Committees}
 
 \todo{TODO: mention scientific coordination?}
 
-\section{SINE2020}
+\section{Software development for data treatment software for the neutron scattering and muon spectroscopy}
 \label{SINE2020}
 
-This section describes guidelines specific to the development of data treatment software for the neutron and muon communities.
```
Comment (AnthonyLim23) [inline guidelines_for_collaborative_development_of_sustainable_data_treatment_software.tex]:
```suggestion
This section describes guidelines specific to the development of data treatment software for the neutron and muon communities.
```

File: guidelines_for_collaborative_development_of_sustainable_data_treatment_software.tex (hunk 15 above)
Comment (AnthonyLim23) [reply guidelines_for_collaborative_development_of_sustainable_data_treatment_software.tex]: did we get an answer to this?

**Pull request merged**